src/lib/components/molecules/tooltip/index.jsx

```diff
@@ -3,6 +3,7 @@ import { useMousePosition } from './useMousePosition'
 import { createPortal } from 'preact/compat'
 
 export function Tooltip({ for: elementRef, renderIn: refOrSelector, children }) {
+
```

Comment:
Would you mind just reverting this change? I think you probably didn't mean to touch the tooltip component.